setup.py

```diff
@@ -75,8 +75,8 @@
     'invenio-utils==0.2.0',  # Not fully Invenio 3 ready
     'invenio>=3.0.0a1,<3.1.0',
     'inspire-crawler~=0.0,>=0.2.7',
+    'inspire-dojson~=0.0,>=0.0.50',
```

Comment:
Version is temporary as https://travis-ci.org/inspirehep/inspire-dojson/builds/248357361 appears to be stuck.